### DISCUSSION TITLE
WebDAV Bug in 4.15.1 Windows

### DISCUSSION THREAD

**Discussion by fluffi80** [iterate-ch/cyberduck#15430] [Q&A]
Folders and files with spaces in their names are not displayed. Can anyone confirm this?

**Comment by ylangisc** [comment]
This is most probably a server side encoding issue. To confirm, please post a [debug log](https://docs.cyberduck.io/mountainduck/support/#debug-log) when browsing a folder containing such files and folders.

**Comment by fluffi80** [comment]
[mountainduck.log](https://github.com/iterate-ch/cyberduck/files/13550720/mountainduck.log)

**Comment by ylangisc** [comment]
The log file contains error entries only. You need to enable [debug log](https://docs.cyberduck.io/mountainduck/support/#debug-log) in Preferences → Connection before reproducing the issue.

**Comment by fluffi80** [comment]
Sorry!
[mountainduck.log](https://github.com/iterate-ch/cyberduck/files/13550954/mountainduck.log)

**Comment by dkocher** [comment]
Refer to our notes on [interoperability with Microsoft IIS](https://docs.mountainduck.io/protocols/webdav/#interoperability-with-microsoft-iis):

* Enable [HEAD method](https://docs.microsoft.com/en-us/iis/manage/configuring-security/configure-request-filtering-in-iis#http-verbs)
* Enable [double escaping](https://docs.microsoft.com/en-us/iis/manage/configuring-security/configure-request-filtering-in-iis#configure-double-escaping)
* Disable [Filter Double-Encoded Requests](https://docs.microsoft.com/en-us/iis/manage/configuring-security/use-request-filtering#filter-double-encoded-requests)

**Reply by fluffi80** [reply]
Thank you. But how and where can I make the appropriate settings in Mountain Duck?
